ui/components/app/name/name-details/name-details.tsx

```diff
@@ -27,11 +27,11 @@ import {
   IconName,
   Label,
   Modal,
-  ModalContent,
-  ModalHeader,
   ModalOverlay,
   Text,
 } from '../../../component-library';
+import { ModalContent } from '../../../component-library/modal-content/deprecated';
+import { ModalHeader } from '../../../component-library/modal-header/deprecated';
 import {
   AlignItems,
   BlockSize,
```

Comment:
No story to easily compare UI but can be confident there are no changes from other examples